tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -42,6 +43,7 @@
 @pytest.fixture(autouse=True)
 def clear_session():
     yield
+    _shut_down_async_logging()
```

Comment:
curious on this - if we delete this line would the test fail? Basically users don't know that they need to call this method.

Reply:
It doesn't fail, there are just some left threads